tox.ini

```diff
@@ -13,6 +14,7 @@ deps =
     django22: Django>=2.2,<3.0
     django30: Django>=3.0,<3.1
     django31: Django>=3.1,<3.2
+    django40: Django>=4.0,<4.1
     mysql: mysqlclient>=1.4.2,<1.5
```

Comment:
is there any new releases of mysqlclient that we could bump??

Reply:
probably. is it breaking anything? as long as it's working for the tests i dont think it's our responsibility to test db client regressions as long as there is nothing broken

Reply:
we could try https://pypi.org/project/mysqlclient/2.1.0/ & see if it pass